app/models/form/sales/pages/buyer1_ethnic_group.rb

```diff
@@ -2,19 +2,21 @@ class Form::Sales::Pages::Buyer1EthnicGroup < ::Form::Page
   def initialize(id, hsh, subsection)
     super
     @id = "buyer_1_ethnic_group"
-    @depends_on = [
-      {
-        "buyer_has_seen_privacy_notice?" => true,
-      },
-      {
-        "buyer_not_interviewed?" => true,
-      },
-    ]
   end
 
   def questions
     @questions ||= [
       Form::Sales::Questions::Buyer1EthnicGroup.new(nil, nil, self),
     ]
   end
+
+  def routed_to?(log, _current_user)
+    super && page_routed_to?(log)
+  end
+
+  def page_routed_to?(log)
+    return false if log.form.start_year_2025_or_later? && log.is_staircase?
+
+    log.buyer_has_seen_privacy_notice? || log.buyer_not_interviewed?
```

Comment:
We still care about 2023 routing, I guess? Otherwise I think buyer_has_seen_privacy_notice is forced to be true for the section to be enabled and these could be simplified, but that wasn't true for 2023.

Reply:
Hmm, good point, I think we care about it while we display 2023 logs, which would be until the end of crossover I think (?)